app/build.gradle

```
@@ -44,9 +44,9 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
 
-    testImplementation 'junit:junit:4.13.2'
-    testImplementation 'com.google.truth:truth:1.1.2'
-    testImplementation 'io.mockk:mockk:1.10.6'
+    testImplementation "junit:junit:$junit_version"
+    testImplementation "com.google.truth:truth:$truth_version"
+    testImplementation "io.mockk:mockk:$mockk_version"
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 }
```

프로그래밍 요구사항 중 2번 'app 모듈은 domain 모듈에 의존해야 한다.'를 빠트리셨네요.
domain 모듈은 추가하셨지만 의존을 하고 있지 않습니다.
app gradle dependencies에 추가해주시기 바랍니다. ✌

```suggestion
dependencies {
    implementation project(":domain")
...
```